requirements.txt

```diff
@@ -0,0 +1 @@
+pylint==1.9.5
```

Comment:
Out of curiosity, is this the last version supporting py2 before they moved to py3-only? A short comment with the reason could be here.

Reply:
Yes, they dropped py2 in 2.0+. I'll add a comment

Reply:
btw there is a nice hack to see  available versions:
```bash
$ pip2 install pylint==    # to see all available versions for py2
$ pip3 install pylint==    # to see all available versions for py2
```